clients/hadoopfs/src/test/java/io/lakefs/LakeFSFileSystemTest.java

```diff
@@ -44,6 +45,7 @@
 import org.junit.Test;
 import org.mockito.Answers;
 import org.mockito.ArgumentCaptor;
+import org.mockito.stubbing.OngoingStubbing;
```

Comment:
```suggestion
```
Please tell IntelliJ organize imports that _it had **one job**_.

Reply:
:) not like goimports - optimize imports